core/libs/commonwealth/commonwealth/utils/commands.py

```diff
@@ -1,7 +1,20 @@
 import subprocess
+from pathlib import Path
 
+from loguru import logger
 
-def run_command(command: str, check: bool = True) -> "subprocess.CompletedProcess['str']":
+
+class KeyNotFound(Exception):
+    """Raised when the SSH key is not found."""
+
+
+def file_exists(file_name: str) -> bool:
+    return Path(file_name).exists()
```

Comment:
This functions appears to be unnecessary to be used in a single place, and without much text to replace.

Reply:
fair enough, I got rid of it.